### ISSUE TITLE
Dynamic warning first

### ISSUE THREAD

**Issue by greglg45** [louislam/uptime-kuma#4543]
### 📑 I have found these related issues/pull requests

no issue

### 🏷️ Feature Request Type

Status-page

### 🔖 Feature description

good morning, 
We currently use uptime kuma to monitor certain elements of our network. We have a screen dedicated to displaying the status page (http://kuma:3001/status/page). 
The problem is that we have a large number of checks on this status page and it is impossible for us to see them at first glance. 

Would it be possible to move the warning elements to the top of the page? 
![Capture d’écran 2024-02-29 092038](https://github.com/louislam/uptime-kuma/assets/98177090/9c74c03b-a85a-43a8-8d31-50761a7bd274)

### ✔️ Solution

add an automatic sorting function to add warnings to the top of the page

**Comment by CommanderStorm** [comment]
Duplicate of https://github.com/louislam/uptime-kuma/issues/1936

TLDR: currently only such a PR has been opened for the dashboard
